rubrik_cdm/api.py

```diff
@@ -70,20 +71,20 @@ def _common_api(self, call_type, api_version, api_endpoint, config=None, job_sta
         if call_type != 'JOB_STATUS':
             self._api_validation(api_version, api_endpoint)
 
-            if '/cluster/me/bootstrap' or '/cluster/me/version' in api_endpoint:
-                if self.ipv6_addr != "":
-                    header = {
-                        'Content-Type': 'application/json',
-                        'Accept': 'application/json',
-                        'Host': '[' + self.ipv6_addr + ']'
-                    }
-                    self.log('Created boostrap header: ' + str(header))
-                else:
-                    header = {
-                        'Content-Type': 'application/json',
-                        'Accept': 'application/json',
-                    }
-                    self.log('Created boostrap header: ' + str(header))
+            # if '/cluster/me/bootstrap' or '/cluster/me/version' in api_endpoint:
+            #     if self.ipv6_addr != "":
+            #         header = {
+            #             'Content-Type': 'application/json',
+            #             'Accept': 'application/json',
+            #             'Host': '[' + self.ipv6_addr + ']'
```

Comment:
@drew-russell If `Host` is not set in the header, IPv6 bootstrap will fail

Reply:
There is a bug in the current implementation. So commenting out (temporarily) to prevent the bigger bug.

 I haven't dug into it much but I don't think `self.ipv6_addr` is defined properly in the `Connect()` class. If you call '/cluster/me/bootstrap' or '/cluster/me/version' or it throws an error. 

Will fix on a subsequent PR.